client/webserver/site/src/js/markets.js

```diff
@@ -246,6 +246,31 @@ export default class MarketsPage extends BasePage {
       page.cancelPass.value = ''
     }
 
+    // Resize elements on screen when the page is resized
+    this.resize = () => {
+      let marketTableHeight
+      const largeScreenCutoffWidth = 992
+      if (window.innerWidth < largeScreenCutoffWidth) {
+        // In smaller screens, the order panel and the market tables are stacked
+        // on top of each other, and we want the market tables to have the same height
+        // as the live orders table.
+        marketTableHeight = page.liveTableContainer.offsetHeight - page.liveTableHead.offsetHeight
+      } else {
+        // On large screens, we keep the market tables the same height as the order panel.
+        // We subtract 10 pixels because if the markets tables becomes taller than the order
+        // panel, then the order panel will resize based on the market tables, and they will
+        // all keep getting taller.
+        marketTableHeight =
+          page.orderPanel.offsetHeight -
+          page.buysTableHeader.offsetHeight -
+          page.buysHeader.offsetHeight -
+          10
+      }
+      page.sellRows.style.height = page.buyRows.style.height = `${marketTableHeight}px`
```

Comment:
I'd expect the table heights would be something for the css and bootstrap responsive design classes to handle automatically, rather than computing height and ending up with something like `<tbody id="buyRows" class="stylish-overflow" style="height: 519px;">`, but I don't really have the solution.  Perhaps @buck54321 or another ui dev with bootstrap knowledge can make a suggestion.

The `largeScreenCutoffWidth = 992` matches a responsive cutoff (`$grid-breakpoints` in application.scss), so it feels like that should come out of the css somehow or just be used by the responsive and flex classes.  

The computed height still seems to be too high in a lot of cases.  e.g. a simnet layout with this PR:

![image](https://user-images.githubusercontent.com/9373513/128399077-99909157-00ce-42b4-a51a-7ea43d46b5fc.png)

The two order tables are making it higher than necessary above.

A similar layout on testnet without these changes:

![image](https://user-images.githubusercontent.com/9373513/128399164-d580be2a-695e-4d93-b43b-a842009365e5.png)

Note the order form not being taller than necessary.